volttrontesting/platform/auth_control_test.py

```diff
@@ -137,3 +167,186 @@ def test_auth_remove(volttron_instance_encrypt):
     entries = auth_list_json(platform)
     assert len(entries) > 0
     assert_auth_entries_same(entries[-1], _auth_entry1.__dict__)
+
+
```

Comment:
So for now it's just so that the user can organize their own intent on how the instance should operate.